src/ata.c

```diff
@@ -389,7 +387,7 @@ void ata_init()
 
 	console_printf("ata: probing devices\n");
 
-	for(i=0;i<4;i++) {
+	for(i=0;i<1;i++) {
```

Comment:
What is this for? I know there were timeout warning messages during boot, but I think we should figure out why `4` was there in the first place before changing it.